src/app/Api.NoSql/Startup.cs

```diff
@@ -133,11 +132,11 @@ private void ConfigureDi(IServiceCollection services)
                 t => t.Name.EndsWith("Service")
             );
 
-            // register services from Common project
+            // register services from Common.Services project
             services.AddTransientByConvention(
                 new List<Type> { typeof(IUserService) },
-                t => t.Namespace.StartsWith("Common.Services.") && t.Name.EndsWith("Service"),
-                t => t.Namespace.StartsWith("Common.Services.") && t.Name.EndsWith("Service")
+                t => t.Namespace.StartsWith("Common.Services.Services.") && t.Name.EndsWith("Service"),
```

Comment:
A suggestion to rename services namespace to `Common.Services.Sql` and `Common.Services.NoSql` respectively. As 'Services.Services' looks odd. Very discussable.